server/helpers/responseHandler.js

```diff
@@ -0,0 +1,68 @@
+const errorHandler = (code, err, res) => {
+  switch (code) {
+    case 401:
+      return res.status(401).json({
+        status: 'fail',
+        message: err
+      });
+    case 404:
+      return res.status(404).json({
+        status: 'fail',
+        message: err
+      });
+    case 403:
+      return res.status(403).json({
+        status: 'fail',
+        message: err
+      });
+    case 406:
+      return res.status(406).json({
+        status: 'fail',
+        message: err
+      });
+    case 409:
+      return res.status(409).json({
+        status: 'fail',
+        message: err
+      });
+    default:
+      return res.status(400).json({
+        status: 'fail',
+        message: err
+      });
+  }
+};
```

Comment:
the body of your error is the same across all error status. Why not assign it to a variable and call the variable. 
```
 const error = { status: 'fail', message: err }
return res.status(400).json(error)
```